ansible/playbooks/roles/opensearch/tasks/install-ops.yml

```diff
@@ -0,0 +1,52 @@
+---
+- name: Download Opensearch
+  include_role:
+    name: download
+    tasks_from: download_file
+  vars:
+    file_name: "{{ item }}"
+  with_items:
+    - "opensearch-{{ versions[ansible_os_family].ops_version }}-linux-x64.tar.gz"
```

Comment:
How about ARM? Maybe better to define name of this file in defaults/main.yml or using `set_fact`?

Reply:
This point is addressed now so the variables describing binaries are reflecting the architecture.